ios/RNZumoKit.m

```diff
@@ -33,11 +33,9 @@ -  (NSDictionary *)decimalLocale
     return [NSDictionary dictionaryWithObject:@"." forKey:NSLocaleDecimalSeparator];
 }
 
-// ZumoKit iOS SDK runs its WebSockets on the main queue. Running other methods 
-// in a separate queue introduces mememory access race conditions and crashes.
 - (dispatch_queue_t)methodQueue
 {
-    return dispatch_get_main_queue();
+    return dispatch_queue_create("money.zumo.zumokit", DISPATCH_QUEUE_SERIAL);
```

Comment:
How come `money.zumo.zumokit` instead of `zumo.money.zumokit`? just curious

Reply:
From https://developer.apple.com/documentation/dispatch/1453030-dispatch_queue_create

> Because applications, libraries, and frameworks can all create their own dispatch queues, a reverse-DNS naming style (com.example.myqueue) is recommended. This parameter is optional and can be NULL.

Reply:
That makes sense! Thanks :)